src/tests/SearchView.test.jsx

```diff
@@ -362,9 +372,13 @@ describe('Search View', () => {
     const searchInput = getByRole('textbox');
 
     await user.type(searchInput, 'coconut');
+    await user.clear(searchInput);
     await user.type(searchInput, 'spam@eggs');
+    await user.clear(searchInput);
     await user.type(searchInput, 'spamspamspamand@eggs.');
+    await user.clear(searchInput);
     await user.type(searchInput, 'iamalmostlongenoughtobeahashbutnotquite');
+    await user.clear(searchInput);
```

Comment:
not clearing search input meant that each value was being appending to searchValue